internal/controller/openslo/slo_controller.go

```diff
@@ -70,7 +70,7 @@ func (r *SLOReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 				log.Error(err, "Failed to update SLO ready status")
 				return ctrl.Result{}, err
 			}
-			return ctrl.Result{}, nil
+			return ctrl.Result{RequeueAfter: time.Second * 5}, nil
```

Comment:
Why do we need to do this?

Reply:
That comes from this previous PR/commit https://github.com/oskoperator/osko/commit/13deb5f8f4717fddbd2957cfcf20868546e89a66#diff-843e6e74bf74fb2516f330ec7542781819a468e33325a9e82465e0524ad980d5R66

Reply:
... and solves the merge conflict that was here before :) check the diff of this branch vs master, this isn't actually changing between those two

Reply:
Hmm, I would say that this must have been one of my brain farts then :D